dotnet/src/dotnetframework/GxClasses/Core/GXUtilsCommon.cs

```diff
@@ -5766,20 +5768,63 @@ public static class GxImageUtil
 
 		private static Bitmap BitmapCreateFromStream(string filePathOrUrl)
 		{
-			using (Stream s = ImageFile(filePathOrUrl).GetStream())
+			if (filePathOrUrl.StartsWith("http://") || filePathOrUrl.StartsWith("https://"))
```

Comment:
Would it be possible to use uri constants like this:
Uri uri;
if (Uri.TryCreate(imageFile, UriKind.Absolute, out uri) && (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps))